notify.go

```diff
@@ -71,14 +71,14 @@ func buildContext(hook hookEvent, ws, logDir string) *grimNotificationContext {
 	return &grimNotificationContext{hook.Owner, hook.Repo, hook.EventName, hook.Target, hook.UserName, ws, logDir}
 }
 
-func notify(config *effectiveConfig, hook hookEvent, ws string, notification grimNotification, logger *log.Logger) error {
+func notify(config *effectiveConfig, hook hookEvent, ws, basename string, notification grimNotification, logger *log.Logger) error {
 	if hook.EventName != "push" && hook.EventName != "pull_request" {
 		return nil
 	}
 
 	ghErr := setRefStatus(config.gitHubToken, hook.Owner, hook.Repo, hook.StatusRef, notification.GithubRefStatus(), "", "")
 
-	logDir := config.resultRoot + "/" + hook.Owner + "/" + hook.Repo
+	logDir := config.resultRoot + "/" + hook.Owner + "/" + hook.Repo + "/" + basename
```

Comment:
This should not be calculated here.  This is duplicating the same work as the "resultsPath" calculation.  If the results path gets changed then this message becomes incorrect.  Remove this line and instead use the resultsPath